build.gradle

```diff
@@ -51,7 +51,7 @@ allprojects {
     group = "io.kotlintest"
 
     if (isTravis) {
-        version "4.0.0." + travisBuildNumber + "-SNAPSHOT"
+        version "4.0." + travisBuildNumber + "-SNAPSHOT"
```

Comment:
Was this intentional?

Reply:
Yes, I shouldn't have set it to 4.0.0 in the first place.